pages/api/statistics/totals.js

```diff
@@ -36,17 +36,14 @@ export async function getTotalStats() {
 
   let activeProfiles = 0;
   try {
-    activeProfiles = await Profile.find({}).estimatedDocumentCount();
+    activeProfiles = await Profile.countDocuments({ isEnabled: true });
   } catch (e) {
     logger.error(e, "failed to load profile statistics");
   }
 
-  const directoryPath = path.join(process.cwd(), "data");
-  let totalProfiles = [];
+  let totalProfiles = 0;
   try {
-    totalProfiles = fs
-      .readdirSync(directoryPath)
-      .filter((item) => item.includes("json"));
+    totalProfiles = await Profile.countDocuments({});
```

Comment:
I think it is more efficient to use the meta data of the collection, than to count the exact number

```suggestion
    totalProfiles = await Profile.find({}).estimatedDocumentCount();
```